test/system/030-run.bats

```diff
@@ -788,6 +788,9 @@ json-file | f
         # :O (overlay) required with rootfs; see #14504
         run_podman run --name=$randomname --rootfs $romount:O echo "Hello world"
         is "$output" "Hello world"
+        if findmnt "$romount" >/dev/null ; then
+            die "'podman run' did not umount $romount"
+        fi
```

Comment:
Seems like this is a false positive because we do podman image mount so it checks the image mount path which is rightfully not unmounted by run. This would instead actually need to ensure the overlay mount is unmounted